app/controllers/api/v1/reports_controller.rb

```diff
@@ -11,7 +12,7 @@ class ReportsController < V1::BaseController
       param :per_page, String, :desc => "number of entries per request"
 
       def index
-        @reports = Report.
+        @reports = ConfigReport.
           authorized(:view_reports).
```

Comment:
Search the codebase and you'll also find DB seeds, fixtures and the dashboard helper need updating.